codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/ServerCodegenVisitor.kt

```diff
@@ -160,6 +161,9 @@ open class ServerCodegenVisitor(
             .let { ModelTransformer.create().copyServiceErrorsToOperations(it, settings.getService(it)) }
             // Add `Box<T>` to recursive shapes as necessary
             .let(RecursiveShapeBoxer::transform)
+            // Refactors constrained member shapes into non-constrained shapes of a new standalone type
+            // that has some constraints.
+            .let(RefactorConstrainedMemberType::transform)
```

Comment:
Have renamed it to `ConstrainedStructureMemberTransform` and have removed the word `refactor` from wherever it was being used.